CMakeLists.txt

```diff
@@ -533,10 +533,7 @@ install_helper(
 #
 
 add_subdirectory(src/nvim)
-# Read compilation flags from src/nvim, used in config subdirectory below.
-include(GetCompileFlags)
-get_compile_flags(NVIM_VERSION_CFLAGS)
```

Comment:
`include(GetCompileFlags)`  is not called anywhere else. After this change, does `nvim --version` still show the git hash? 

On Travis build log, it shows `v0.4.0-dev` . (But that also happens on master, wtf?)

```
+bin/nvim --version
NVIM v0.4.0-dev
Build type: Debug
```